### PR TITLE
Post views implementation

### PR DESCRIPTION
**Pull request by amogiska**

### CODE REVIEW

File: crumbs/crumbs/User.swift [hunk 1]
```diff
@@ -32,6 +33,7 @@ class User {
         self.karma = snapshot.get("karma") as! Int
         self.views = snapshot.get("views") as! Int
         self.likedPostIds = snapshot.get("liked_posts") as! [DocumentReference]
+        self.viewedPostIds = snapshot!.get("viewed_posts") as! [DocumentReference]
```
Comment (senkevinli) [inline crumbs/crumbs/User.swift:36]:
```suggestion
        self.viewedPostIds = snapshot.get("viewed_posts") as! [DocumentReference]
```

File: crumbs/crumbs/PostViewController.swift [hunk 2]
```diff
@@ -174,6 +174,13 @@ class PostViewController: UIViewController, UITableViewDelegate, UITableViewData
             self.postImage = post.uiImage
         }
         
+        if (!CUR_USER.hasViewedPost(p: self.post)){
+            print("got here")
```
Comment (senkevinli) [inline crumbs/crumbs/PostViewController.swift:178]:
```suggestion
```

File: crumbs/crumbs/PostViewController.swift [hunk 3]
```diff
@@ -174,6 +174,13 @@ class PostViewController: UIViewController, UITableViewDelegate, UITableViewData
...
+            print(self.post.docRef)
```
Comment (senkevinli) [inline crumbs/crumbs/PostViewController.swift:179]:
```suggestion
```

**Review by senkevinli — CHANGES_REQUESTED** [review]
Posts should now start with 0 views instead of 1.

**Review by philolin — APPROVED** [review]

**Pull request merged**